deploy/kubernetes/elasticsearch/elasticsearch.gcloud.yaml

```diff
@@ -14,10 +14,6 @@ spec:
       - secretName: elasticsearch-secrets
     fileRealm:
       - secretName: elasticsearch-secrets
-  http:
-    tls:
-      selfSignedCertificate:
-        disabled: true
```

Comment:
how is seqr gonna have access to ES with this change. I don't see any code changes that would cause seqr to use the generated certificate?

Reply:
Still working on this -- you can ignore this PR for now, still in draft status :)

Reply:
the short version, is that we'd need to add the CA generated by ECK as a secret for kibana, and then update the kibana configuration to use it for verification, and switch to an https url.